examples/fscloud/main.tf

```diff
@@ -0,0 +1,118 @@
+  key_map                   = { "icd-pg" = ["${var.prefix}-pg"] }
+}
+
+# Create IAM Access Policy to allow Key protect to access Postgres instance
+resource "ibm_iam_authorization_policy" "policy" {
+  source_service_name         = "databases-for-postgresql"
+  source_resource_group_id    = module.resource_group.resource_group_id
+  target_service_name         = "kms"
+  target_resource_instance_id = module.key_protect_all_inclusive.key_protect_guid
+  roles                       = ["Reader"]
+}
+
+##############################################################################
+# Get Cloud Account ID
+##############################################################################
+
+data "ibm_iam_account_settings" "iam_account_settings" {
+}
+
+##############################################################################
+# VPC
+##############################################################################
+resource "ibm_is_vpc" "example_vpc" {
+  name           = "${var.prefix}-vpc"
+  resource_group = module.resource_group.resource_group_id
+  tags           = var.resource_tags
+}
+
+resource "ibm_is_subnet" "testacc_subnet" {
+  name                     = "${var.prefix}-subnet"
+  vpc                      = ibm_is_vpc.example_vpc.id
+  zone                     = "${var.region}-1"
+  total_ipv4_address_count = 256
+  resource_group           = module.resource_group.resource_group_id
+}
+
+##############################################################################
+# Create CBR Zone
+##############################################################################
+module "cbr_zone" {
+  source           = "git::https://github.com/terraform-ibm-modules/terraform-ibm-cbr//cbr-zone-module?ref=v1.1.2"
+  name             = "${var.prefix}-VPC-network-zone"
+  zone_description = "CBR Network zone representing VPC"
+  account_id       = data.ibm_iam_account_settings.iam_account_settings.account_id
+  addresses = [{
+    type  = "vpc", # to bind a specific vpc to the zone
+    value = ibm_is_vpc.example_vpc.crn,
+  }]
+}
+
+##############################################################################
+# Postgres Instance
+##############################################################################
+
+module "postgresql_db" {
+  source              = "../../profiles/fscloud"
+  resource_group_id   = module.resource_group.resource_group_id
+  name                = "${var.prefix}-postgres"
+  region              = var.region
+  pg_version          = var.pg_version
+  key_protect_key_crn = module.key_protect_all_inclusive.keys["icd-pg.${var.prefix}-pg"].crn
+  resource_tags       = var.resource_tags
+  allowlist           = var.allowlist
+  cbr_rules = [
+    {
+      description      = "${var.prefix}-postgres access only from vpc"
+      enforcement_mode = "enabled" #Postgresql does not support report mode
+      account_id       = data.ibm_iam_account_settings.iam_account_settings.account_id
+      tags = [
```

Comment:
From yesterdays deep dive we will completely remove tags from CBRs

Reply:
As we are removing tags completely, then we need to remove this from variable type format too....is it right?

Reply:
Made the changes.